FFMpegCore.Examples/Program.cs

```diff
@@ -79,7 +79,8 @@ await FFMpegArguments
     FFMpeg.PosterWithAudio(inputPath, inputAudioPath, outputPath);
     // or 
 #pragma warning disable CA1416
-    using var image = Image.FromFile(inputImagePath);
+    using var image = Image.FromFile(inputImagePath);    // Using FFMpegCore.Extensions.System.Drawing.Common
+    //using var image = SKBitmap.Decode(inputImagePath); // Using FFMpegCore.Extensions.SkiaSharp
```

Comment:
Uncomment and rename to skiaImage or similar? Please avoid commented out code :)
You could also add a second block for the skia example, to avoid confusion